scalopus_tracing/src/lttng/ctfevent.cpp

```diff
@@ -50,43 +50,78 @@ CTFEvent::CTFEvent(std::string line)
   // Extract the timestamp, its between []
   size_t start_pos = line.find("[");
   size_t end_pos = line.find("]");
+  if (start_pos == std::string::npos || end_pos == std::string::npos)
+  {
+    return;
+  }
   std::string timestamp_str = line.substr(start_pos + 1, end_pos - 1);
-  timestamp_ = std::stod(timestamp_str);
+  try
+  {
+    timestamp_ = std::stod(timestamp_str);
+  }
+  catch (const std::invalid_argument& e)
```

Comment:
```suggestion
  catch (const std::invalid_argument&)
```
Or print or re-`throw` the exception error message.

Reply:
We don't have access to a logger function from this object, and I don't just want to start spamming `std::cerr`. I'll remove the variable for the exception.